packages/x-date-pickers/src/DatePicker/DatePicker.types.ts

```diff
@@ -54,6 +54,11 @@ export interface DatePickerProps<
    * @default 4 on desktop, 3 on mobile
    */
   yearsPerRow?: 3 | 4;
+  /**
+   * If `true`, the popover or modal will close after submitting the full date.
+   * @default `true` for desktop, `false` for mobile variant (based on the chosen wrapper and `desktopModeMediaQuery` prop).
```

Comment:
Not fan of just adding `variant` after `mobile` here, I would stick with the old wording or use variants for both:


```suggestion
   * @default `true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop).
```

or

```suggestion
   * @default `true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop).
```

Reply:
We currently call them "variants" [in the doc](https://mui.com/x/react-date-pickers/date-picker/#available-components) as well. 🙈 
P.S. Both of your suggestions seem to be the same. 🙈

Reply:
Suggestion updated :laughing: 
My problem is not using the word "variant", it's the phrasing "true for desktoop, false for mobile variant" which seems inconsistent to me.